virt/support/virt-troubleshooting.adoc

```diff
@@ -6,6 +6,22 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+[id="diagnostics_{context}"]
+== Diagnostics
```

Comment:
I think you might be overdocumenting the feature. I recommend that you replace this section with a single sentence: "You can view the status of virtual machines (VMs), network, storage, and snapshots on the xref:...[Diagnostics tab] of the web console."

Please add a line to the UI table on the Support overview assembly (virt/support/virt-support-overview.html) with a link:

|VirtualMachines → VirtualMachine → VirtualMachine details → Diagnostics tab
|VM status conditions and volume snapshot status

(I will check the other lines of that table to see whether any of the tabs have moved to the new VM tab).

Reply:
Thanks for the feedback. Again, the build won't be successful yet unfortunately so here are some screenshots of the changes requested. Let me know if anything needs fixing.

<details>
<summary>Screenshots</summary>

`virt-support-overview`
![image](https://user-images.githubusercontent.com/119876664/232504137-2f69a85a-f575-4747-b5ee-c83bbf3d553b.png)

`virt-troubleshooting`
![image](https://user-images.githubusercontent.com/119876664/232504337-4487c1bb-6de9-43aa-9e39-5382ea93d651.png)
  
</details>